tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -1122,371 +1122,15 @@ When using per-instance uniforms, there are some restrictions you should be awar
 Built-in variables
 ------------------
 
-A large number of built-in variables are available, like ``UV``, ``COLOR`` and ``VERTEX``. What variables are available depends on the type of shader (``spatial``, ``canvas_item`` or ``particle``) and the function used (``vertex``, ``fragment`` or ``light``).
-For a list of the built-in variables that are available, please see the corresponding pages:
+A large number of built-in variables are available, like ``UV``, ``COLOR`` and
+``VERTEX``. What variables are available depends on the type of shader
+(``spatial``, ``canvas_item``, ``particle``, etc) and the
+function used (``vertex``, ``fragment``, ``light``, ``start``, ``process,
+``sky``, or ``fog``). For a list of the built-in variables that are available,
+please see the corresponding pages:
 
 - :ref:`Spatial shaders <doc_spatial_shader>`
 - :ref:`Canvas item shaders <doc_canvas_item_shader>`
 - :ref:`Particle shaders <doc_particle_shader>`
 - :ref:`Sky shaders <doc_sky_shader>`
 - :ref:`Fog shaders <doc_fog_shader>`
-
-Built-in functions
-------------------
-
-A large number of built-in functions are supported, conforming to GLSL ES 3.0.
-When vec_type (float), vec_int_type, vec_uint_type, vec_bool_type nomenclature
-is used, it can be scalar or vector.
```

Comment:
I think its worth keeping this section and only having one sentence that simply points to the new page. Users may have a hard time finding it otherwise